index.php

```diff
@@ -1,2 +1,60 @@
 <?php
-# Silence is golden.
+if (!defined('ABSPATH')) { // Exit if accessed directly
+	exit;
+}
+require_once 'vendor/autoload.php';
```

Comment:
I haven't tested, we should be able to now.  All I wanted in the PR was to switch to WordPress AJAX and get rid of `wp-load.php`.  With this change done we should be able to remove `vendor/autoload.php` and move ALL class instantiations to cloudflare.php.